spec/migrations/20150311181430_add_draft_to_orchestration_templates_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_migration
+
+describe AddDraftToOrchestrationTemplates do
+  let(:orchestration_template_stub) { migration_stub(:OrchestrationTemplate) }
+
+  migration_context :up do
+    it "sets false to draft column" do
+      orchestration_template_stub.create!
+      orchestration_template_stub.create!
```

Comment:
@jrafanie I just wanted to test it for more records,  but yes I think that  it is not needed.

updated, thanks!